apis/v1alpha1/validation/validation.go

```diff
@@ -34,56 +30,6 @@ var (
 	}
 )
 
-// ValidateGateway validates gw according to the Gateway API specification.
-// For additional details of the Gateway spec, refer to:
-//  https://gateway-api.sigs.k8s.io/spec/#networking.x-k8s.io/v1alpha1.Gateway
-func ValidateGateway(gw *gatewayv1a1.Gateway) field.ErrorList {
-	return validateGatewaySpec(&gw.Spec, field.NewPath("spec"))
-}
```

Comment:
Interesting. So this is the only validation we have today.
@robscott @youngnick Should we keep this function around and simply `return nil` here to keep the other machinery around?
Else, this will require a change in the webhook manifest as well. And if users don't change the webhook definition then they will get errors.

Reply:
A dummy function may be better.

Reply:
Hi @hbagdi and @tokers, I just added the dummy functions. Let me know your feedback.

Reply:
So, I agree that we should keep the dummy function, btu put a pointer to the issue and this PR in the docstring, along with an explanation like:
```go
// This function is currently empty as the validations have been moved to CRD field validations.
// It's expected that further validation will be added back here in the future.
```

Reply:
Thanks @youngnick. I just updated the code with your comment.

Reply:
Thanks a lot. Just fixed